crates/cubecl-linalg/src/matmul/kernels/error.rs

```diff
@@ -165,3 +179,17 @@ impl Debug for MatmulAvailabilityError {
         }
     }
 }
+
+pub enum MatmulUnimplementedError {
+    Quantization,
+}
+
+impl Debug for MatmulUnimplementedError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            MatmulUnimplementedError::Quantization => {
+                writeln!(f, "Quantization is not yet available.")
```

Comment:
When combined with `MatmulLaunchError`, the message will read as 

```
Unable to launch matmul because the feature is not ready: Quantization is not yet available.
```

This is a bit redundant.

Maybe we don't need this one and simply pass a string to `MatmulLaunchError::Unimplemented`.